<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Joey J. Gardner </title> <meta name="author" content="Joey J. Gardner"> <meta name="description" content="Publications in reverse chronological order."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon.png?f7dc3dc56dd72ce24a337043a401015c"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://insilijo.github.io/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Joey</span> J. Gardner </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/books/">bookshelf </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">Publications in reverse chronological order.</p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="gardner2025" class="col-sm-8"> <div class="title">From Microbiota to Microbiome: Understanding our Greater Selves</div> <div class="author"> </div> <div class="periodical"> <em>Metabolon</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.metabolon.com/blog/from-microbiota-to-microbiome-understanding-our-greater-selves/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Blog</a> </div> <div class="abstract hidden"> <p>The human microbiome is becoming more and more essential to our understanding of biology and health outcomes. This blog outlines how metabolomics can uniquely interrogate the dynamic, fundamental interactions between host and microbiota at scale.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="gardner2024" class="col-sm-8"> <div class="title">Commercial Opportunities in Multiomics</div> <div class="author"> </div> <div class="periodical"> <em>Metabolon</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.metabolon.com/blog/commercial-opportunities-in-multiomics/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Blog</a> </div> <div class="abstract hidden"> <p>Multiomics is poised to open new, translational paths for a diversity of researchers and research goals. This blog investigates the utility of these approaches from a commercial perspective and how metabolomics, in particular, is key to them.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="gardner2023investigating" class="col-sm-8"> <div class="title">Investigating the unique ability of <i>Trichodesmium</i> to fix carbon and nitrogen simultaneously using MiMoSA</div> <div class="author"> Joseph J Gardner, Bri-Mathias S Hodge, and Nanette R Boyle </div> <div class="periodical"> <em>mSystems</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://journals.asm.org/doi/full/10.1128/msystems.00601-20" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>The open ocean is an extremely competitive environment, partially due to the dearth of nutrients. <i>Trichodesmium erythraeum</i>, a marine diazotrophic cyanobacterium, is a keystone species in the ocean due to its ability to fix nitrogen and leak 30 to 50% into the surrounding environment, providing a valuable source of a necessary macronutrient to other species. While there are other diazotrophic cyanobacteria that play an important role in the marine nitrogen cycle, <i>Trichodesmium</i> is unique in its ability to fix both carbon and nitrogen simultaneously during the day without the use of specialized cells called heterocysts to protect nitrogenase from oxygen. Here, we use the advanced modeling framework called <b>m</b>ult<b>i</b>scale <b>m</b>ulti<b>o</b>bjective <b>s</b>ystems <b>a</b>nalysis (MiMoSA) to investigate how <i>Trichodesmium erythraeum</i> can reduce dimolecular nitrogen to ammonium in the presence of oxygen. Our simulations indicate that nitrogenase inhibition is best modeled as Michealis-Menten competitive inhibition and that cells along the filament maintain microaerobia using high flux through Mehler reactions in order to protect nitrogenase from oxygen. We also examined the effect of location on metabolic flux and found that cells at the end of filaments operate in distinctly different metabolic modes than internal cells despite both operating in a photoautotrophic mode. These results give us important insight into how this species is able to operate photosynthesis and nitrogen fixation simultaneously, giving it a distinct advantage over other diazotrophic cyanobacteria because they can harvest light directly to fuel the energy demand of nitrogen fixation.<br> <b>Importance</b> <i>Trichodesmium erythraeum</i> is a marine cyanobacterium responsible for approximately half of all biologically fixed nitrogen, making it an integral part of the global nitrogen cycle. Interestingly, unlike other nitrogen-fixing cyanobacteria, <i>Trichodesmium</i> does not use temporal or spatial separation to protect nitrogenase from oxygen poisoning; instead, it operates photosynthesis and nitrogen fixation reactions simultaneously during the day. Unfortunately, the exact mechanism the cells utilize to operate carbon and nitrogen fixation simultaneously is unknown. Here, we use an advanced metabolic modeling framework to investigate and identify the most likely mechanisms <i>Trichodesmium</i> uses to protect nitrogenase from oxygen. The model predicts that cells operate in a microaerobic mode, using both respiratory and Mehler reactions to dramatically reduce intracellular oxygen concentrations.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="gardner2019m" class="col-sm-8"> <div class="title"> <b>M</b>ult<b>i</b>scale <b>M</b>ulti<b>o</b>bjective <b>S</b>ystems <b>A</b>nalysis (MiMoSA): an advanced metabolic modeling framework for complex systems</div> <div class="author"> Joseph J Gardner, Bri-Mathias S Hodge, and Nanette R Boyle </div> <div class="periodical"> <em>Scientific reports</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/s41598-019-53188-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>In natural environments, cells live in complex communities and experience a high degree of heterogeneity internally and in the environment. Even in ‘ideal’ laboratory environments, cells can experience a high degree of heterogeneity in their environments. Unfortunately, most of the metabolic modeling approaches that are currently used assume ideal conditions and that each cell is identical, limiting their application to pure cultures in well-mixed vessels. Here we describe our development of Multiscale Multiobjective Systems Analysis (MiMoSA), a metabolic modeling approach that can track individual cells in both space and time, track the diffusion of nutrients and light and the interaction of cells with each other and the environment. As a proof-of concept study, we used MiMoSA to model the growth of <i>Trichodesmium erythraeum</i>, a filamentous diazotrophic cyanobacterium which has cells with two distinct metabolic modes. The use of MiMoSA significantly improves our ability to predictively model metabolic changes and phenotype in more complex cell cultures.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="gardner2019multiscale" class="col-sm-8"> <div class="title">Multiscale, Multiparadigm Metabolic Modeling of the Keystone Diazotrophic Cyanobacterium, <i>Trichodesmium erythraeum</i> </div> <div class="author"> Joseph Jay Gardner </div> <div class="periodical"> <em>Colorado School of Mines</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.proquest.com/openview/2faf7910aa6a0be802fa4f226ad424e4/1?pq-origsite=gscholar&amp;cbl=18750&amp;diss=y" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Earth is a crowded place; studies estimate between 2 and 12 million species exist on earth, with new organisms being formally described every day. This biodiversity is a result of organisms being forced to cohabitate in innately competitive environments; proximity and nutrient limitation require organisms to interact. Current modeling techniques neglect many of these phenomena, considering cells as separate entities, homogeneous populations, or static members of a population. Biological data conflict with these models: organisms are highly dynamic even in the most ideal growth scenarios. Moreover, they operate multilaterally, optimizing many responses to their environments, needs, and other cells, reflecting multiobjective biological strategies for which there is no current elegant mathematical explanation. Expanding these computational techniques to better capture biology will help reduce the solution space of experimentation, widen research focuses, characterize existing ecosystems, and apply better predictions.<br> <br> This work addresses some of the fundamental shortcomings of current metabolic modeling techniques while characterizing a crucial diazotrophic cyanobacterium, <i>Trichodesmium erythraeum</i>. It uses genome-scale modeling approaches to characterize a unique metabolism at the center of the carbon, nitrogen, and phosphorus biogeochemical cycles. It ties these models to experimentation, both personally conducted and from literature, anchoring the findings to biological reality and determining the needs of the field. It progresses to generate a MultIscale MultiObjective Systems Analysis (MIMOSA) framework that allows cells to self-govern within the context of a community and environment, creating emergent cell behavior and better illustrating the dynamic procedures of individual cells. Finally, it employs these models to interrogate the unique nitrogen fixation capabilities of <i>T. erythraeum</i>, presenting hypotheses on its productivity as it integrates carbon fixation, a circadian cycle, and microaerobia to operate efficiently. The work is a step toward distilling actionable information from a plethora of resources in a significant microorganism.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="gardner2017use" class="col-sm-8"> <div class="title">The use of genome-scale metabolic network reconstruction to predict fluxes and equilibrium composition of N-fixing versus C-fixing cells in a diazotrophic cyanobacterium, <i>Trichodesmium erythraeum</i> </div> <div class="author"> Joseph J Gardner and Nanette R Boyle </div> <div class="periodical"> <em>BMC systems biology</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://link.springer.com/article/10.1186/s12918-016-0383-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p><strong>Background</strong><br> Computational, genome based predictions of organism phenotypes has enhanced the ability to investigate the biological phenomena that help organisms survive and respond to their environments. In this study, we have created the first genome-scale metabolic network reconstruction of the nitrogen fixing cyanobacterium T. erythraeum and used genome-scale modeling approaches to investigate carbon and nitrogen fluxes as well as growth and equilibrium population composition.<br> <br> <strong>Results</strong><br> We created a genome-scale reconstruction of <i>T. erythraeum</i> with 971 reactions, 986 metabolites, and 647 unique genes. We then used data from previous studies as well as our own laboratory data to establish a biomass equation and two distinct submodels that correspond to the two cell types formed by T. erythraeum. We then use flux balance analysis and flux variability analysis to generate predictions for how metabolism is distributed to account for the unique productivity of T. erythraeum. Finally, we used in situ data to constrain the model, infer time dependent population compositions and metabolite production using dynamic Flux Balance Analysis. We find that our model predicts equilibrium compositions similar to laboratory measurements, approximately 15.5% diazotrophs for our model versus 10-20% diazotrophs reported in literature. We also found that equilibrium was the most efficient mode of growth and that equilibrium was stoichiometrically mediated. Moreover, the model predicts that nitrogen leakage is an essential condition of optimality for T. erythraeum; cells leak approximately 29.4% total fixed nitrogen when growing at the optimal growth rate, which agrees with values observed in situ.<br> <br> <strong>Conclusion</strong><br> The genome-metabolic network reconstruction allows us to use constraints based modeling approaches to predict growth and optimal cellular composition in <i>T. erythraeum</i> colonies. Our predictions match both in situ and laboratory data, indicating that stoichiometry of metabolic reactions plays a large role in the differentiation and composition of different cell types. In order to realize the full potential of the model, advance modeling techniques which account for interactions between colonies, the environment and surrounding species need to be developed.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="aucoin2016omics" class="col-sm-8"> <div class="title">Omics in <i>Chlamydomonas</i> for biofuel production</div> <div class="author"> Hanna R Aucoin, <em>Joseph Gardner</em>, and Nanette R Boyle </div> <div class="periodical"> <em>In Lipids in plant and algae development</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://link.springer.com/chapter/10.1007/978-3-319-25979-6_18" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>In response to demands for sustainable domestic fuel sources, research into biofuels has become increasingly important. Many challenges face biofuels in their effort to replace petroleum fuels, but rational strain engineering of algae and photosynthetic organisms offers a great deal of promise. For decades, mutations and stress responses in photosynthetic microbiota were seen to result in production of exciting high-energy fuel molecules, giving hope but minor capability for design. However, ‘-omics’ techniques for visualizing entire cell processing has clarified biosynthesis and regulatory networks. Investigation into the promising production behaviors of the model organism <i>C. reinhardtii</i> and its mutants with these powerful techniques has improved predictability and understanding of the diverse, complex interactions within photosynthetic organisms. This new equipment has created an exciting new frontier for high-throughput, predictable engineering of photosynthetically produced carbon-neutral biofuels.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Joey J. Gardner. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?c15de51d4bb57887caa2c21988d97279"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>